.vscode/spellright.dict

```diff
@@ -0,0 +1,3 @@
+postgres
```

Comment:
@ebgitelman try to avoid committing files that aren't part of the work you're doing. You can add the names of files like this to your local .gitignore if that makes it easier